components/admin-panel/sections/ActivityLog/index.js

```diff
@@ -200,6 +205,7 @@ const ActivityLog = ({ accountSlug }) => {
   const { data, loading, error } = useQuery(activityLogQuery, {
     variables: getQueryVariables(accountSlug, router),
     context: API_V2_CONTEXT,
+    fetchPolicy: 'network-only',
```

Comment:
@Betree : I found that without this the `CollectivePickerAsync` doesn't get updated property. I am not exactly sure why this is the case yet. 🤔

Reply:
I'm good with that, it makes sense to always use the data available for something like this.